### DISCUSSION TITLE
Changing from TALOS to ET Open in Standalone SO 2.3.2

### DISCUSSION THREAD

**Discussion by McSh4dow** [Security-Onion-Solutions/securityonion#2059] [Unsupported Versions] [answered]
Hello,

I was just wondering if I could change the ruleset from TALOS to another.

In the docs I found that I have to modify the /opt/so/saltstack/local/pillar/minions/<manager.sls> to change the ruleset.
ETPro:
idstools:
  config:
    ruleset: 'ETPRO'
    oinkcode: 'MYOINKCODE'

What about ETOpen? Does ruleset have to be 'ETOpen' and oinkcode blank?

Thank you in advance!

**Comment by weslambert** [comment, marked as answer]
Yes, that is correct.

**Reply by weslambert** [reply]
ETOPEN, blank oinkcode
